.idea/workspace.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this (I would recommend adding .idea to global gitignore)

Reply:
oops 
and you already said it!
removed (and also added .idea to global gitignore)